Logger/LogFactory.cs

```diff
@@ -2,10 +2,23 @@
 {
     public class LogFactory
     {
+        private string logFilePath;
```

Comment:
Fields should be named with underscore PascalCase
```suggestion
        private string _LogFilePath;
```